src/styles/utilis/reset.scss

```
@@ -0,0 +1,7 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-self: center;
```

```suggestion
  justify-content: center;
```
This will center the whole calendar in the page.